frontend/src/app/testResults/TestResultPrintModal.scss

```diff
@@ -1,26 +1,45 @@
-.sr-result-print-buttons {
-  display: flex;
-  justify-content: flex-end;
-
-  & button {
-    margin: 0 0 0 1.5rem;
-  }
-}
-
 @media print {
   body {
-    .dont-print {
+    // ensures extra content is not printed in results page
+    &.ReactModal__Body--open {
```

Comment:
⚠️ If I don't have this the SimpleReport test results print page has a blank page that appears